var/spack/repos/builtin/packages/py-cppy/package.py

```diff
@@ -14,7 +14,11 @@ class PyCppy(PythonPackage):
 
     maintainers = ["iarspider"]
 
+    version("1.2.1", sha256="83b43bf17b1085ac15c5debdb42154f138b928234b21447358981f69d0d6fe1b")
     version("1.1.0", sha256="4eda6f1952054a270f32dc11df7c5e24b259a09fddf7bfaa5f33df9fb4a29642")
 
+    depends_on("python@3.7:", when="@1.2:", type=("build", "run"))
     depends_on("python@3:", type=("build", "run"))
+    depends_on("py-setuptools@42:", when="@1.2:", type="build")
     depends_on("py-setuptools", type="build")
+    depends_on("py-setuptools-scm+toml@3.4.3:", when="@1.2:", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools-scm+toml@3.4.3: +toml", when="@1.2:", type="build")
```

Reply:
`+toml` would be added twice then. Did you miss the one before the `@`?

Reply:
@manuelakuhn yup. Sorry!